backend/tracim_backend/config.py

```diff
@@ -51,6 +53,18 @@
 DEPOT_CONTENT_CONF_PREFIX = "uploaded_files"
 
 
+class TranslationProvider(Enum):
```

Comment:
Wouldn't using a simple dictionary be enough here?
Something like:
```python
TRANSLATION_SERVICE_CLASSES = {
  "systran": SystranTranslationService
}
```
BTW I think this definition should be moved in the translation .py file.

Reply:
I think i found something in between which is better with both dictionnary and enum.